tests/actor/test_battery_status.py

```diff
@@ -190,21 +190,30 @@ class Message(Generic[T]):
 class FakeSelect:
     """Helper class to mock Select object used in BatteryStatusTracker"""
 
-    def __init__(
+    # This is just number of Select instance attributes.
+    def __init__(  # pylint: disable=too-many-arguments
         self,
         battery: Optional[BatteryData] = None,
         inverter: Optional[InverterData] = None,
         request_result: Optional[SetPowerResult] = None,
+        battery_timer: bool = False,
+        inverter_timer: bool = False,
     ) -> None:
         """Create FakeSelect instance
 
         Args:
             battery: Expected battery message. Defaults to None.
             inverter: Expected inverter message. Defaults to None.
+            battery_timer: If true battery data timer will be set to indicate that no
+                message was received since a while.
+            inverter_timer: If tru inverter data timer will be set to indicate that no
```

Comment:
typo in `tru` and I'd rephase it to something like `if true the inverter data timer will be set to indicate that no messages have been received for a while`.

Also you might want to rename both variables to indicate they are flags rather than timers, for instance, `set_battery_timer` or `battery_timer_flag`

Reply:
Done